docker/compose/docker-compose.yml

```diff
@@ -79,8 +79,11 @@ services:
     ports:
       - "12345:12345"
     command: segmentstore
+    privileged: true
     environment:
       WAIT_FOR: bookie1:3181,bookie2:3182,bookie3:3183,hdfs:8020
```

Comment:
I have the same question here about fixing the number of bookies. What if I want to reply with a larger BK cluster? In fact, what if I have already a BK cluster that I want to use here? Would that be possible at all?